editor/py/serve.py

```diff
@@ -272,6 +331,7 @@ def compare(file_name):
   remote_ascii = text_format.MessageToString(remote)
   local_ascii = text_format.MessageToString(local)
   if remote_ascii != local_ascii:
+    print(local_ascii)
```

Comment:
More debugging messages?

Reply:
Conor, I saw you resolved this discussion. Does that mean you prefer to keep the print statements? Or am I misunderstanding our PR workflow?

Reply:
Sorry; I resolved it since I removed it in a new commit and assumed it was minor enough you didn't need to re-review the code.